larky/src/test/resources/vendor_tests/luhn/luhn_test.star

```diff
@@ -0,0 +1,26 @@
+load("@vendor//luhn", luhn="luhn")
```

Comment:
lets sort this (as per "pep8") and have this after `load("@vendor//asserts", "asserts")`

Reply:
https://github.com/OmegaPointZero/starlarky/blob/bc58a870acf4683607ee81934ee0ab7538a8ad8e/larky/src/test/resources/vendor_tests/luhn/luhn_test.star#L1-L3